packages/core/admin/admin/src/content-manager/components/BlocksEditor/Toolbar/index.js

```diff
@@ -624,7 +615,7 @@ const BlocksToolbar = ({ disabled }) => {
 
   return (
     <Toolbar.Root aria-disabled={disabled} asChild>
-      {/* Remove after the RTE Blocks Beta release (paddingRight and width) */}
+      {/* Remove after the RTE Blocks Stable release (paddingRight and width) */}
       <ToolbarWrapper gap={2} padding={2} paddingRight={4} width="100%">
```

Comment:
I believe we should remove this comment as well as width and paddingRight on ToolbarWrapper